features/mfa.feature

```diff
@@ -145,6 +145,32 @@ Feature: Prompt for MFA credentials
       |           |   TOTP   | , backup codes | does not support WebAuthn |   TOTP           |
       |           |          |   backup codes | does not support WebAuthn |      backup code |
 
+
+  Scenario: Defaulting to the manager despite having a used mfa
+    Given I provide credentials that have a manager and a used TOTP
+      And the user's browser supports WebAuthn
+    When I login
+    Then I should see a prompt for a manager rescue code
+
+  Scenario: Defaulting to a more recently used webauth mfa
+    Given I provide credentials that have a used WebAuthn and TOTP
+      And the user's browser supports WebAuthn
+    When I login
+    Then I should see a prompt for a WebAuthn
```

Comment:
The steps of this scenario don't seem to specify which MFA option was used most recently. Perhaps we could add a line like "And I have used WebAuthn most recently" (or whichever type) to specify that.

The test scenarios for each of the different MFA types will actually be similar enough that we could probably use a Scenario Outline and a list of Examples to make it easier to read the test (i.e. so that the differences are more apparent, rather than hiding in mostly-duplicated Scenarios).

Just as a possible example...
```gherkin
Scenario Outline: Defaulting to the most recently used (non-emergency) MFA type
Given I provide credentials that have WebAuthn, TOTP, backup codes, and a manager email
  And my browser supports WebAuthn
  And I have used <most recent MFA type> most recently
When I login
Then I should see a prompt for a <initial MFA prompt>

Examples:
  | most recent MFA type | initial MFA prompt |

  # Allowed default:
  | WebAuthn             | WebAuthn           |
  | TOTP                 | TOTP               |
  | backup code          | backup code        |

  # Disallowed default:
  | manager code         | WebAuthn           |
```
You could even add a middle column to the Examples section for "supports WebAuthn or not" to include that factor in this same test, if you want to. Or it could be a separate test more focused on just that detail.

Some of the other tests will probably need some refinement once this change is implemented, since the desired behavior is now different than when they were written... maybe just an "And I haven't used any MFA option yet" if we want to keep their changes to a minimum.